roles/openshift_rhv/defaults/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Defaults for oVirt/RHV provider integration
+compatibility_version: 4.2
+data_center_name: Default
+wait_for_ip: true
+openshift_rhv_cluster: Default
+template_name: openshift_node
+
+# oVirt VM Profiles
+openshift_rhv_vm_profile:
+  master:
+    cluster: "{{ openshift_rhv_cluster }}"
+    template: "{{ template_name }}"
+    memory: 16GiB
+    cores: 2
+    high_availability: true
+    disks:
+    - size: 15GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: docker_disk
+      interface: virtio
+    - size: 30GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: localvol_disk
+      interface: virtio
+    - size: 25GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: etcd_disk
+      interface: virtio
+    state: running
+  node:
+    cluster: "{{ openshift_rhv_cluster }}"
+    template: "{{ template_name }}"
+    memory: 8GiB
+    cores: 2
+    high_availability: true
+    disks:
+    - size: 15GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: docker_disk
+      interface: virtio
+    - size: 30GiB
+      storage_domain: "{{ openshift_rhv_data_store }}"
+      name: localvol_disk
+      interface: virtio
+    state: running
+
+openshift_rhv_vm_manifest:
+- name: 'master'
```

Comment:
This is example and as such you can change it with `sed` in one simple shot.

Reply:
I disagree...
Defaults are meant so that the playbook will be able to run with no special intervention.
I think using sed is a workaround and not really how users would like to work.
You expect the users to know your playbook and start messing with it... 

If you give them a parameter like "vm_name_prefix" with default like openshift.
And add a comment that this in a name they need to choose to identify their OpenShift installation(It can be a staging/prod/test/ customers name etc.)and  will allow the creation of different vms for several OpenShift installations
will make it more structured and less work for the users.

Reply:
This is do-able, could you provide a patch? Any variables we add should be prefixed with `openshift_rhv_` I would have the default as the empty string.

Reply:
Yes. I'll can do that.